libs/utils/WordPressUtils/src/main/java/org/wordpress/android/util/JSONUtils.java

```diff
@@ -213,6 +213,11 @@ public static boolean getBool(JSONObject json, String name) {
         return true;
     }
 
+    public static boolean isStringTrue(JSONObject json, String name) {
```

Comment:
The `isStringTrue()` routine still exists - I think it can be deleted now. Also, should `getBool()` be updated to check for `no` (since your routine originally checked for `yes`)?